change_logs/release_v0.14.0.md

```diff
@@ -38,7 +38,7 @@ Also I'd like to take this opportunity to recognize and thank a few folks that h
 * [Eldad Assis](https://github.com/eldada)
 * [Tobias](https://github.com/mycrEEpy)
 * [Helge Sychla](https://github.com/hsychla)
-* [Markusi75](https://github.com/Makusi75)
+* [Makusi75](https://github.com/Makusi75)
```

Comment:
Nope - This is correct!

Reply:
You sure?

![image](https://user-images.githubusercontent.com/2119212/115476283-97cdfb00-a20f-11eb-944e-f4a9a61c06f3.png)

Reply:
Sorry - indeed - you're correct. Tx Josh!!

Reply:
Fwiw, this sort of thing is not as uncommon as you might think. Sometimes I'm careful to call it out. Often people respond exactly as you did. I'm sorry.